tests/projects/test_entry_point.py

```diff
@@ -41,12 +49,15 @@ def test_entry_point_compute_command():
     with TempDir() as tmp:
         storage_dir = tmp.path()
         command = entry_point.compute_command({"name": "friend", "excitement": 10}, storage_dir)
+        assert command == "python greeter.py hi friend --goodbye --excitement 10"
+        command = entry_point.compute_command({"name": "friend", "excitement": 10, "args": ''},
+                                              storage_dir)
         assert command == "python greeter.py hi friend --excitement 10"
         with pytest.raises(ExecutionException):
             entry_point.compute_command({}, storage_dir)
         # Test shell escaping
         name_value = "friend; echo 'hi'"
-        command = entry_point.compute_command({"name": name_value}, storage_dir)
+        command = entry_point.compute_command({"name": name_value, "args": ""}, storage_dir)
```

Comment:
Would be good to either add in a case or modify one of the existing tests where `args` has a value in it (e.g. `--goodbye`).